src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_RANDOM_VALUE = 100;
+    public static final int NUMBER_FIGURES = 5;
```

Comment:
```suggestion
    public static final int MAX_RANDOM_VALUE = 100;
    public static final int NUMBER_FIGURES = 5;
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```

Reply:
use this fields instead of constructions like `new Random()` or `new ColorSupplier()`